### PR TITLE
fix watch generator uses different name pattern from gen project

### LINKED ISSUE #362: Some kubernetes objects do not implement IKubernetesObject
I have found some Kubernetes objects which do not implement IKubernetesObject:
- Extensionsv1beta1Ingress
- Networkingv1beta1Ingress
I use the Kubernetes client  version 1.6.16-gb07e78afa4 
Is this on purpose or a bug?

### PR DESCRIPTION
**Pull request by tg123**
fix #362 

also fix some watch operations are missing

### CODE REVIEW

File: gen/KubernetesWatchGenerator/Program.cs [hunk 1]
```diff
@@ -150,7 +101,7 @@ static void ToXmlDoc(RenderContext context, IList<object> arguments, IDictionary
                     {
                         if (!first)
                         {
-                            context.Write(Environment.NewLine);
+                            context.Write("\n");
```
Comment (brendandburns) [inline gen/KubernetesWatchGenerator/Program.cs]: Shouldn't we keep the `Environment.NewLine` ?

**Comment by brendandburns** [comment]
Looks good, one really minor comment.

File: gen/KubernetesWatchGenerator/Program.cs (hunk 1 above)
Comment (tg123) [reply gen/KubernetesWatchGenerator/Program.cs]: it should be `eol` of the generated files, `IKubernetes.Watch.cs`, `Kubernetes.Watch.cs` and `ModelExtensions.cs`.

those files are generated using `lf` already.
and should be generated using `lf` only as `.editorconfig` defined the rule `eol = lf`

Comment (brendandburns) [reply gen/KubernetesWatchGenerator/Program.cs]: ok, thanks.

**Pull request merged**